modules/navier_stokes/doc/content/source/fv/kernels/INSFVBoussinesqBodyForce.md

```diff
@@ -0,0 +1,15 @@
+# INSFVBoussinesqBodyForce
+
+This object adds a $\rho\alpha\vec{g}(T - T_{ref})$ term to the incompressible
+Navier Stokes (INS) equations where $\rho$ is the density, $\alpha$ is the thermal
+expansion coefficient, $\vec{g}$ is the gravity vector, $T$ is the temperature,
+and $T_{ref}$ is a reference temperature. The term above introduces the
+Boussinesq approximation into the INS equations, which allows for modeling
+natural convection. This object must be added for every component
+of the velocity vector, e.g. $u$, $v$, $w$.
```

Comment:
I don't think you need it for u and v if g and w are aligned

Reply:
:+1: